src/gap.c

```diff
@@ -10,110 +10,40 @@
-#include <src/precord.h>                /* plain records */
-
-#include <src/lists.h>                  /* generic lists */
-#include <src/listoper.h>               /* operations for generic lists */
-#include <src/listfunc.h>               /* functions for generic lists */
-#include <src/plist.h>                  /* plain lists */
-#include <src/set.h>                    /* plain sets */
-#include <src/vector.h>                 /* functions for plain vectors */
-#include <src/vecffe.h>                 /* functions for fin field vectors */
-#include <src/blister.h>                /* boolean lists */
-#include <src/range.h>                  /* ranges */
-#include <src/stringobj.h>              /* strings */
-#include <src/vecgf2.h>                 /* functions for GF2 vectors */
-#include <src/vec8bit.h>             /* functions for other compressed
-                                           GF(q) vectors                   */
-#include <src/objfgelm.h>               /* objects of free groups */
-#include <src/objpcgel.h>               /* objects of polycyclic groups */
-#include <src/objscoll.h>               /* single collector */
-#include <src/objccoll.h>               /* combinatorial collector */
-#include <src/objcftl.h>                /* from the left collect */
-
-#include <src/dt.h>                     /* deep thought */
-#include <src/dteval.h>                 /* deep thought evaluation */
-
-#include <src/sctable.h>                /* structure constant table */
-#include <src/costab.h>                 /* coset table */
-#include <src/tietze.h>                 /* tietze helper functions */
-
-#include <src/code.h>                   /* coder */
-
-#include <src/exprs.h>                  /* expressions */
-#include <src/stats.h>                  /* statements */
-#include <src/funcs.h>                  /* functions */
-
-#include <src/intrprtr.h>               /* interpreter */
-
-#include <src/compiler.h>               /* compiler */
-
-#include <src/compstat.h>               /* statically linked modules */
-
-#include <src/saveload.h>               /* saving and loading */
-
-#include <src/streams.h>                /* streams package */
-#include <src/sysfiles.h>               /* file input/output */
-#include <src/weakptr.h>                /* weak pointers */
-#include <src/profile.h>                /* profiling */
-#include <src/hookintrprtr.h>
+#include <src/gap.h>
 
+#include <src/ariths.h>
+#include <src/bool.h>
+#include <src/calls.h>
+#include <src/code.h>
+#include <src/compiler.h>
+#include <src/compstat.h>
+#include <src/exprs.h>
+#include <src/funcs.h>
 #include <src/gapstate.h>
-
-#include <src/objset.h>
+#include <src/gvars.h>
+#include <src/integer.h>
+#include <src/lists.h>
+#include <src/opers.h>
+#include <src/plist.h>
+#include <src/precord.h>
+#include <src/records.h>
+#include <src/read.h>
+#include <src/saveload.h>
+#include <src/stats.h>    // for ClearError
```

Comment:
Also here, why a comment?

Reply:
Should be: ˋFIXME: only for ClearErrorˋ. Point is, ClearError really should be somewhere else, several files include stats.h just for it.